src/client.ts

```diff
@@ -206,6 +206,33 @@ export class Client {
   geolocate(request: GeolocateRequest): Promise<GeolocateResponse> {
     return geolocate(request, this.axiosInstance);
   }
+  /**
+   * An example use of this function.
+   *
+   * ```javascript
+   * import { Client } from '@googlemaps/google-maps-services-js';
+   * 
+   * const getFirstResult = async (theAddress) => {
+   *   const args = {
+   *     params: {
+   *       key: '<your-api-key>',
+   *       address: theAddress,
+   *     }
+   *   };
+   *   const client = new Client();
+   *   const result = await client.geocode(args);
+   *   if (result.status === 200) {
```

Comment:
this check shouldn't be necessary as an adapter exists to map this to the actual result

Reply:
see https://github.com/googlemaps/google-maps-services-js/blob/master/src/adapter.ts